build.rs

```diff
@@ -585,18 +605,19 @@ fn nasm(file: &Path, arch: &str, include_dir: &Path, out_dir: &Path) {
         .arg(include_dir)
         .arg("-Xgnu")
         .arg("-gcv8")
-        .arg(file);
+        .arg(c_root_dir.join(file));
```

Comment:
There are still two relative paths here, in `Command::new` and the `include/`, so I think this probably won't work if the current working directory isn't what Cargo would set, unless you are building from a packaged release of ring that skips the `generate_sources_and_preassemble` step. 

If you actually are running nasm in Bazel through build.rs, I am guessing you need a solution for the `Command::new` independently of the current working directory issue. Probably we should introduce a cargo cfg directive for providing the path to nasm.